tests/test_data.py

```diff
@@ -0,0 +1,85 @@
+import os
+import json
+import pytest
+from jsonschema import validate, ValidationError
+
+from bods_validate import this_dir, format_checker, absolute_path_to_schema_dir, resolver
+from bods_validate import MissingStatementTypeError
+from bods_validate import bods_validate_package, bods_validate_statement
+
+
+@pytest.mark.parametrize('json_path', [
+    'data/entity-statement/valid/valid-entity-statement.json',
+    'data/person-statement/valid/valid-person-statement.json',
+    'data/beneficial-ownership-statement/valid/valid-beneficial-ownership-statement.json',
+    pytest.mark.xfail('../examples/flat-serialisation/gb-coh-entity-statement.json'),
```

Comment:
This example should be fixed in latest commit, so we'd expect it to pass now.

Reply:
I've rebased onto 0.1-dev, but this still seems to fail.

Reply:
Ah, thanks, that should be fixed now.